flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/api/java/internal/StreamTableEnvironmentImpl.java

```diff
@@ -335,6 +335,12 @@ protected boolean isEagerOperationTranslation() {
 		return true;
 	}
 
+	@Override
+	public String explain(boolean extended) {
+		// throw exception directly, because the operations to explain are always empty
+		throw new TableException("'explain' method is unsupported in StreamTableEnvironment.");
```

Comment:
`String explain(Table table)` can work in `StreamTableEnvironment`. This comment is misleading.

Reply:
Yes. We can improve the message.